spec/game_turn_spec.rb

```diff
@@ -0,0 +1,163 @@
+module TicTacToe
+  require "game_turn"
+  require "board"
+  require "validation"
+  require "eval_game"
```

Comment:
You can use a spec_helper file to require the classes/modules you need and then you will only need to require the spec_helper file in each of your test files.